### PR TITLE
multiply int8/uint8 for AVX512

### PR DESCRIPTION
**Pull request by mikekgfb**
Summary: multiply int8/uint8 for AVX512

Test Plan: sandcastle, github

Differential Revision: D52393918

cc @jgong5 @mingfeima @XiaobingSuper @sanchitintel @ashokei @jingxu10

### CODE REVIEW

File: aten/src/ATen/cpu/vec/vec512/vec512_int.h [hunk 1]
```diff
@@ -1002,13 +1002,37 @@ Vectorized<T> inline int_elementwise_binary_512(const Vectorized<T>& a, const Ve
 template <>
 Vectorized<int8_t> inline operator*(const Vectorized<int8_t>& a, const Vectorized<int8_t>& b) {
   // We don't have an instruction for multiplying int8_t
```
Comment (sanchitintel) [inline aten/src/ATen/cpu/vec/vec512/vec512_int.h:1004]: May we remove the comment `We don't have an instruction for multiplying int8_t`? Thanks!

**Review by jgong5 — APPROVED** [review]
LGTM

**Comment by malfet** [comment]
Sorry for jumping in late, but why no unit tests for this one?

**Comment by mikekgfb** [comment]
> Sorry for jumping in late, but why no unit tests/benchmarks for this one? (Considering that AVX512 can sometimes be slower than AVX2)

This is for users that already have picked AVX512 - if they don't, they end up in AVX2 land (upgrade as separate PR n#[116235](https://github.com/pytorch/pytorch/pull/116235). 

I started writing unit tests, but found that we already have pre-existing unit tests for testing the various operations in vec.h and it's derived children.

**Pull request closed without merging**